DC-322-L-1 Web Engineering Lab/main.py

```diff
@@ -0,0 +1,174 @@
+import utils
+
+
+def registerAccount():
```

Comment:
Make sure to follow consistency, python supoorts single and double quotes for string. But adhere to use any of it as a sign of clean and consistent code. 

- Either use '' or "", all across your code.
- Use @ as a sign of readability in docstring
- mention parameters & return type in your functions. (as in example below!)
- You mostly mentioned return in docstring as none or None, be consistent, and follow what's suppoorted in python, its 'None'.

A sample docstring can be like this:
def sum(a: int, b: int) -> int:
    ''' 
        @param a: first number
        @param b: second number

        @desc: sum of two numbers

        @return: sum of a and b
    '''
    return a + b

def printSum(a: int, b: int) -> None:
    ''' 
        @param a: first number
        @param b: second number    

        @description: print sum of two numbers
        @return: None
    '''
    print(sum(a, b))